### DISCUSSION TITLE
profiles_sample_rate and traces quota

### DISCUSSION THREAD

**Discussion by dantebarba** [getsentry/sentry-python#2420] [Q&A] [answered]
Hi everyone. I'd like to ask how does the `profiles_sample_rate` count against the performance quota. I understand that every transaction sent by sentry is counted and that is managed by `traces_sample_rate`. So I basically set a low `traces_sample_rate` for production (0.1) but I'm a little bit confused by the `profiles_sample_rate`. 

1. Does `profiles_sample_rate` it count against the performance quota?
2. What does profiles_sample_rate actually do? Does activating it send more transactions to Sentry?

Thanks

**Comment by antonpirker** [comment, marked as answer]
Hey!

1. No `profiles_sample_rate` does not count against the performance quote. Profiles are attached to a performance unit (we call it `transaction`.) You pay for performance unit, no matter if a profile is attached or not. 
2. See this discussion on how the `profiles_sample_rate` works: https://github.com/getsentry/sentry-python/discussions/3039
